### PR TITLE
Move buf price estimation server-side

### PR DESCRIPTION
**Pull request by bufdev**
Some TODOs in here. This relies on the core PR.

### CODE REVIEW

File: private/buf/cmd/buf/command/beta/price/price.go [hunk 1]
```diff
@@ -144,46 +118,27 @@ func run(
...
 		}
-		statsSlice[i] = stats
+		request.Modules[i] = protoModule
 	}
-	tmpl, err := template.New("tmpl").Parse(tmplCopy)
+	service := connectclient.Make(
+		clientConfig,
+		// TODO: there must be some constant here somewhere
+		// We only want to hit the public one for this function
+		"buf.build",
```
Comment (mfridman) [inline private/buf/cmd/buf/command/beta/price/price.go:135]:
```suggestion
		bufconnect.DefaultRemote,
```

Comment (bufdev) [reply private/buf/cmd/buf/command/beta/price/price.go:135]: Commandeer this PR, no need for suggestions lol :-)

Comment (mfridman) [reply private/buf/cmd/buf/command/beta/price/price.go:135]: sure

**Review by mfridman — APPROVED** [review]

**Comment by bufdev** [comment]
Don't merge until the backend is done obv

**Pull request closed without merging**